index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/CSS/style.css">
+</head>
+<body>
+  <div class="container">
+    <header>
+       <nav class="top-nav">
+            <h3 class="logo">My log</h3>
+          <ul>
+              <li>
+                <img class="vector" src="./images/Img/Union.png" alt="Union">
+              </li>
+          </ul> 
+       </nav>
+<!-- Main header section -->
+        <main>
+           <h1 class="greeting">
+            I'm Michael.
+            Glad to see you!
+           </h1>
+    
+           <article>
+              <p class="intro">
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project 
+                you need coded, don’t hesitate to contact me.
+              </p>
+           </article>
+        </main>
```

Comment:
According to the project requirements, this section should be wrapped in a `section` tag and should be aligned using the `flex` display property, please fix this 🙏